webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/LogDisplayer.scala

```diff
@@ -217,6 +219,7 @@ case class ReportLine (
 
       JsObj(
           ( "executionDate", executionDate.toString("yyyy-MM-dd HH:mm:ss") )
+        , ( "runDate", runDate.toString("yyyy-MM-dd HH:mm:ss") )
```

Comment:
Can you use either ISODateTimeFormater .dateTime or our date formatter service with "getDisplayDate" ? We need at least timezone and it's more consistant that way